src/parser/README.md

```diff
@@ -14,6 +14,10 @@ Variable names of the AST usually starts with `n` (for node). This is also histo
 Most files in this directory are generated from a grammar for sablecc3 ( http://www.sablecc.org ).
 In order to generate nit parser, you need the alternate SableCC3 generator ( http://www.mare.ee/indrek/sablecc/ ).
 
+Moreover, the Nit language description file extends the standard SableCC3 syntax and includes parametric productions to factorize the description of the language.
+
+In order to simplify the development of the Nit tools, files produced by the parser generator are committed in the repository; therefore Java and SableCC3 is only required for developers that need to alter the Nit syntax.
```

Comment:
The last sentence could be simplified to `...is only required to alter the Nit syntax.`